### DISCUSSION TITLE
Compra de terrenos de clínicas abortistas en OVR

### DISCUSSION THREAD

**Discussion by IBaixauli** [NGDAOorg/provida#4] [Ideas]
Propongo la compra de terrenos de las clínicas abortistas más importantes del planeta en beneficio de proyectos provida

**Comment by ruth-ga** [comment]
Si existen tantos centros abortistas en el mundo ..., es porque ha habido una inmensa inversión; dicha inversión es muy rentable porque ahora ya sabemos que es un producto utilizado como moneda de cambio... tristemente es el negocio del siglo. Para revertir eso... se necesitaría el doble o más de lo que invirtieron ESOS GRANDES CAPITALISTAS, cuyos nombres ya sabemos....ES UNA MAGNIFICA IDEA.... pero la veo utópica.... Cuánto costaría cada uno de esos ..... Centros... (poq se tendrían que negociar  como propiedades, no como terrenos... )  y de dónde se sacaría el dinero?? cuánto tiempo tardaríamos en juntar una mínima parte...... Habrán filántropos que se unirían a la causa?

**Reply by IBaixauli** [reply]
En OVR, una clínica completa del tamaño del congreso de los Diputados en Madrid, podría costar unos 20 dólares, pero quizá comprando sólo una parte, unos 3 dólares, sería suficiente para dar el golpe de efecto y comenzar a obtener rentabilidad.
